*** Starting uWSGI 2.0.20 (64bit) on [Sun Jun  5 13:47:12 2022] ***
compiled with version: 7.5.0 on 29 May 2022 17:23:40
os: Linux-4.4.0-1128-aws #142-Ubuntu SMP Fri Apr 16 12:42:33 UTC 2021
nodename: goorm
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 8
current working directory: /workspace/mbittest
writing pidfile to /workspace/mbittest/tmp/MBIT.pid
detected binary path: /workspace/mbittest/venv/bin/uwsgi
chdir() to /workspace/mbittest/
your memory page size is 4096 bytes
detected max file descriptor number: 1024
building mime-types dictionary from file /etc/mime.types...554 entry found
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /workspace/mbittest/tmp/MBIT.sock fd 3
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.7.4 (default, Nov  4 2020, 10:17:35)  [GCC 7.4.0]
PEP 405 virtualenv detected: /workspace/mbittest/venv/
Set PythonHome to /workspace/mbittest/venv/
Python main interpreter initialized at 0x55d41ab81140
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 154032 bytes (150 KB) for 1 cores
*** Operational MODE: single process ***
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x55d41ab81140 pid: 565 (default app)
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
spawned uWSGI master process (pid: 565)
spawned uWSGI worker 1 (pid: 570, cores: 1)
subprocess 568 exited with code 0
[pid: 570|app: 0|req: 1/1] 172.17.0.1 () {58 vars in 4158 bytes} [Sun Jun  5 13:47:25 2022] GET / => generated 2276 bytes in 272 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
Not Found: /favicon.ico
[pid: 570|app: 0|req: 2/2] 172.17.0.1 () {54 vars in 4058 bytes} [Sun Jun  5 13:47:25 2022] GET /favicon.ico => generated 2809 bytes in 26 msecs (HTTP/1.1 404) 5 headers in 159 bytes (1 switches on core 0)
[pid: 570|app: 0|req: 3/3] 172.17.0.1 () {60 vars in 4143 bytes} [Sun Jun  5 13:48:34 2022] GET / => generated 2228 bytes in 5 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 570|app: 0|req: 4/4] 172.17.0.1 () {60 vars in 4143 bytes} [Sun Jun  5 13:55:00 2022] GET / => generated 2228 bytes in 5 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 570|app: 0|req: 5/5] 172.17.0.1 () {58 vars in 4135 bytes} [Sun Jun  5 13:55:03 2022] GET /form/ => generated 35263 bytes in 230 msecs (HTTP/1.1 200) 7 headers in 350 bytes (1 switches on core 0)
Not Found: /form/result
[pid: 570|app: 0|req: 6/6] 172.17.0.1 () {56 vars in 4088 bytes} [Sun Jun  5 13:57:46 2022] GET /form/result => generated 2809 bytes in 7 msecs (HTTP/1.1 404) 5 headers in 159 bytes (1 switches on core 0)
Not Found: /form/result/2
[pid: 570|app: 0|req: 7/7] 172.17.0.1 () {56 vars in 4092 bytes} [Sun Jun  5 13:58:02 2022] GET /form/result/2 => generated 2815 bytes in 8 msecs (HTTP/1.1 404) 5 headers in 159 bytes (1 switches on core 0)
[pid: 570|app: 0|req: 8/8] 172.17.0.1 () {56 vars in 4082 bytes} [Sun Jun  5 13:58:15 2022] GET /result/2 => generated 7052 bytes in 7 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
SIGINT/SIGTERM received...killing workers...
worker 1 buried after 1 seconds
goodbye to uWSGI.
VACUUM: pidfile removed.
VACUUM: unix socket /workspace/mbittest/tmp/MBIT.sock removed.
*** Starting uWSGI 2.0.20 (64bit) on [Sun Jun  5 16:42:03 2022] ***
compiled with version: 7.5.0 on 29 May 2022 17:23:40
os: Linux-4.4.0-1128-aws #142-Ubuntu SMP Fri Apr 16 12:42:33 UTC 2021
nodename: goorm
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 8
current working directory: /workspace/mbittest
writing pidfile to /workspace/mbittest/tmp/MBIT.pid
detected binary path: /workspace/mbittest/venv/bin/uwsgi
chdir() to /workspace/mbittest/
your memory page size is 4096 bytes
detected max file descriptor number: 1024
building mime-types dictionary from file /etc/mime.types...554 entry found
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /workspace/mbittest/tmp/MBIT.sock fd 3
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.7.4 (default, Nov  4 2020, 10:17:35)  [GCC 7.4.0]
PEP 405 virtualenv detected: /workspace/mbittest/venv/
Set PythonHome to /workspace/mbittest/venv/
Python main interpreter initialized at 0x55cb97e58db0
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 154032 bytes (150 KB) for 1 cores
*** Operational MODE: single process ***
WSGI app 0 (mountpoint='') ready in 2 seconds on interpreter 0x55cb97e58db0 pid: 5186 (default app)
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
spawned uWSGI master process (pid: 5186)
spawned uWSGI worker 1 (pid: 5195, cores: 1)
subprocess 5187 exited with code 0
[pid: 5195|app: 0|req: 1/1] 172.17.0.1 () {58 vars in 3990 bytes} [Sun Jun  5 16:42:15 2022] GET /result/2 => generated 7052 bytes in 102 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 5195|app: 0|req: 2/2] 172.17.0.1 () {56 vars in 3953 bytes} [Sun Jun  5 16:42:20 2022] GET /admin => generated 0 bytes in 1 msecs (HTTP/1.1 301) 5 headers in 175 bytes (1 switches on core 0)
[pid: 5195|app: 0|req: 3/3] 172.17.0.1 () {56 vars in 3955 bytes} [Sun Jun  5 16:42:20 2022] GET /admin/ => generated 0 bytes in 18 msecs (HTTP/1.1 302) 9 headers in 331 bytes (1 switches on core 0)
[pid: 5195|app: 0|req: 4/4] 172.17.0.1 () {56 vars in 3992 bytes} [Sun Jun  5 16:42:20 2022] GET /admin/login/?next=/admin/ => generated 2214 bytes in 33 msecs (HTTP/1.1 200) 9 headers in 461 bytes (1 switches on core 0)
[pid: 5195|app: 0|req: 5/5] 172.17.0.1 () {66 vars in 4274 bytes} [Sun Jun  5 16:42:22 2022] POST /admin/login/?next=/admin/ => generated 0 bytes in 518 msecs (HTTP/1.1 302) 11 headers in 625 bytes (1 switches on core 0)
[pid: 5195|app: 0|req: 6/6] 172.17.0.1 () {60 vars in 4114 bytes} [Sun Jun  5 16:42:22 2022] GET /admin/ => generated 7256 bytes in 79 msecs (HTTP/1.1 200) 8 headers in 293 bytes (1 switches on core 0)
[pid: 5195|app: 0|req: 7/7] 172.17.0.1 () {58 vars in 4094 bytes} [Sun Jun  5 16:42:28 2022] GET /admin/main/developer/ => generated 7623 bytes in 214 msecs (HTTP/1.1 200) 9 headers in 461 bytes (1 switches on core 0)
[pid: 5195|app: 0|req: 8/8] 172.17.0.1 () {54 vars in 3898 bytes} [Sun Jun  5 16:42:28 2022] GET /admin/jsi18n/ => generated 3195 bytes in 13 msecs (HTTP/1.1 200) 6 headers in 189 bytes (1 switches on core 0)
[pid: 5195|app: 0|req: 9/9] 172.17.0.1 () {58 vars in 4127 bytes} [Sun Jun  5 16:42:30 2022] GET /admin/main/developer/5/change/ => generated 7423 bytes in 69 msecs (HTTP/1.1 200) 9 headers in 461 bytes (1 switches on core 0)
[pid: 5195|app: 0|req: 10/10] 172.17.0.1 () {54 vars in 3907 bytes} [Sun Jun  5 16:42:30 2022] GET /admin/jsi18n/ => generated 3195 bytes in 6 msecs (HTTP/1.1 200) 6 headers in 189 bytes (1 switches on core 0)
[pid: 5195|app: 0|req: 11/11] 172.17.0.1 () {58 vars in 4118 bytes} [Sun Jun  5 16:42:35 2022] GET /admin/main/developer/ => generated 7623 bytes in 209 msecs (HTTP/1.1 200) 9 headers in 461 bytes (1 switches on core 0)
[pid: 5195|app: 0|req: 12/12] 172.17.0.1 () {54 vars in 3898 bytes} [Sun Jun  5 16:42:35 2022] GET /admin/jsi18n/ => generated 3195 bytes in 10 msecs (HTTP/1.1 200) 6 headers in 189 bytes (1 switches on core 0)
SIGINT/SIGTERM received...killing workers...
worker 1 buried after 1 seconds
goodbye to uWSGI.
VACUUM: pidfile removed.
VACUUM: unix socket /workspace/mbittest/tmp/MBIT.sock removed.
*** Starting uWSGI 2.0.20 (64bit) on [Sun Jun  5 17:15:56 2022] ***
compiled with version: 7.5.0 on 29 May 2022 17:23:40
os: Linux-4.4.0-1128-aws #142-Ubuntu SMP Fri Apr 16 12:42:33 UTC 2021
nodename: goorm
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 8
current working directory: /workspace/mbittest
writing pidfile to /workspace/mbittest/tmp/MBIT.pid
detected binary path: /workspace/mbittest/venv/bin/uwsgi
chdir() to /workspace/mbittest/
your memory page size is 4096 bytes
detected max file descriptor number: 1024
building mime-types dictionary from file /etc/mime.types...554 entry found
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /workspace/mbittest/tmp/MBIT.sock fd 3
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.7.4 (default, Nov  4 2020, 10:17:35)  [GCC 7.4.0]
PEP 405 virtualenv detected: /workspace/mbittest/venv/
Set PythonHome to /workspace/mbittest/venv/
Python main interpreter initialized at 0x556b6cdb6db0
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 154032 bytes (150 KB) for 1 cores
*** Operational MODE: single process ***
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x556b6cdb6db0 pid: 6209 (default app)
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
spawned uWSGI master process (pid: 6209)
spawned uWSGI worker 1 (pid: 6212, cores: 1)
subprocess 6210 exited with code 0
[pid: 6212|app: 0|req: 1/1] 172.17.0.1 () {60 vars in 4126 bytes} [Sun Jun  5 17:16:04 2022] GET /admin/main/developer/ => generated 7586 bytes in 474 msecs (HTTP/1.1 200) 9 headers in 461 bytes (1 switches on core 0)
[pid: 6212|app: 0|req: 2/2] 172.17.0.1 () {54 vars in 3875 bytes} [Sun Jun  5 17:16:05 2022] GET /admin/jsi18n/ => generated 3195 bytes in 6 msecs (HTTP/1.1 200) 6 headers in 189 bytes (1 switches on core 0)
Internal Server Error: /result/2
Traceback (most recent call last):
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 470, in parse
    compile_func = self.tags[command]
KeyError: 'endfor'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./main/views.py", line 56, in result
    return render(request, 'main/result.html', context)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/shortcuts.py", line 19, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/backends/django.py", line 34, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/engine.py", line 143, in get_template
    template, origin = self.find_template(template_name)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/engine.py", line 125, in find_template
    template = loader.get_template(name, skip=skip)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/loaders/base.py", line 30, in get_template
    contents, origin, origin.template_name, self.engine,
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 155, in __init__
    self.nodelist = self.compile_nodelist()
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 193, in compile_nodelist
    return parser.parse()
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 478, in parse
    raise self.error(token, e)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 476, in parse
    compiled_result = compile_func(self, token)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/loader_tags.py", line 278, in do_extends
    nodelist = parser.parse()
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 478, in parse
    raise self.error(token, e)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 476, in parse
    compiled_result = compile_func(self, token)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/loader_tags.py", line 216, in do_block
    nodelist = parser.parse(('endblock',))
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 472, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 528, in invalid_block_tag
    get_text_list(["'%s'" % p for p in parse_until], 'or'),
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 37: 'endfor', expected 'endblock'. Did you forget to register or load this tag?
[pid: 6212|app: 0|req: 3/3] 172.17.0.1 () {56 vars in 3980 bytes} [Sun Jun  5 17:16:15 2022] GET /result/2 => generated 173494 bytes in 371 msecs (HTTP/1.1 500) 6 headers in 187 bytes (1 switches on core 0)
Not Found: /result.2
[pid: 6212|app: 0|req: 4/4] 172.17.0.1 () {56 vars in 3980 bytes} [Sun Jun  5 17:16:19 2022] GET /result.2 => generated 2800 bytes in 7 msecs (HTTP/1.1 404) 5 headers in 159 bytes (1 switches on core 0)
Not Found: /result
[pid: 6212|app: 0|req: 5/5] 172.17.0.1 () {56 vars in 3976 bytes} [Sun Jun  5 17:16:23 2022] GET /result => generated 2794 bytes in 9 msecs (HTTP/1.1 404) 5 headers in 159 bytes (1 switches on core 0)
[pid: 6212|app: 0|req: 6/6] 172.17.0.1 () {56 vars in 3964 bytes} [Sun Jun  5 17:16:44 2022] GET / => generated 2193 bytes in 5 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 6212|app: 0|req: 7/7] 172.17.0.1 () {58 vars in 4033 bytes} [Sun Jun  5 17:16:46 2022] GET /form/ => generated 34961 bytes in 230 msecs (HTTP/1.1 200) 7 headers in 350 bytes (1 switches on core 0)
[pid: 6212|app: 0|req: 8/8] 172.17.0.1 () {66 vars in 4217 bytes} [Sun Jun  5 17:17:09 2022] POST /submit/ => generated 0 bytes in 77 msecs (HTTP/1.1 302) 6 headers in 188 bytes (1 switches on core 0)
Internal Server Error: /result/2
Traceback (most recent call last):
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 470, in parse
    compile_func = self.tags[command]
KeyError: 'endfor'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./main/views.py", line 56, in result
    return render(request, 'main/result.html', context)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/shortcuts.py", line 19, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/backends/django.py", line 34, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/engine.py", line 143, in get_template
    template, origin = self.find_template(template_name)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/engine.py", line 125, in find_template
    template = loader.get_template(name, skip=skip)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/loaders/base.py", line 30, in get_template
    contents, origin, origin.template_name, self.engine,
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 155, in __init__
    self.nodelist = self.compile_nodelist()
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 193, in compile_nodelist
    return parser.parse()
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 478, in parse
    raise self.error(token, e)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 476, in parse
    compiled_result = compile_func(self, token)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/loader_tags.py", line 278, in do_extends
    nodelist = parser.parse()
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 478, in parse
    raise self.error(token, e)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 476, in parse
    compiled_result = compile_func(self, token)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/loader_tags.py", line 216, in do_block
    nodelist = parser.parse(('endblock',))
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 472, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 528, in invalid_block_tag
    get_text_list(["'%s'" % p for p in parse_until], 'or'),
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 37: 'endfor', expected 'endblock'. Did you forget to register or load this tag?
[pid: 6212|app: 0|req: 9/9] 172.17.0.1 () {60 vars in 4052 bytes} [Sun Jun  5 17:17:09 2022] GET /result/2 => generated 173638 bytes in 248 msecs (HTTP/1.1 500) 6 headers in 187 bytes (1 switches on core 0)
Internal Server Error: /result/2
Traceback (most recent call last):
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 470, in parse
    compile_func = self.tags[command]
KeyError: 'endfor'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./main/views.py", line 56, in result
    return render(request, 'main/result.html', context)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/shortcuts.py", line 19, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/backends/django.py", line 34, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/engine.py", line 143, in get_template
    template, origin = self.find_template(template_name)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/engine.py", line 125, in find_template
    template = loader.get_template(name, skip=skip)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/loaders/base.py", line 30, in get_template
    contents, origin, origin.template_name, self.engine,
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 155, in __init__
    self.nodelist = self.compile_nodelist()
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 193, in compile_nodelist
    return parser.parse()
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 478, in parse
    raise self.error(token, e)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 476, in parse
    compiled_result = compile_func(self, token)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/loader_tags.py", line 278, in do_extends
    nodelist = parser.parse()
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 478, in parse
    raise self.error(token, e)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 476, in parse
    compiled_result = compile_func(self, token)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/loader_tags.py", line 216, in do_block
    nodelist = parser.parse(('endblock',))
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 472, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 528, in invalid_block_tag
    get_text_list(["'%s'" % p for p in parse_until], 'or'),
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 37: 'endfor', expected 'endblock'. Did you forget to register or load this tag?
[pid: 6212|app: 0|req: 10/10] 172.17.0.1 () {60 vars in 4052 bytes} [Sun Jun  5 17:19:22 2022] GET /result/2 => generated 173638 bytes in 310 msecs (HTTP/1.1 500) 6 headers in 187 bytes (1 switches on core 0)
Internal Server Error: /result/2
Traceback (most recent call last):
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 470, in parse
    compile_func = self.tags[command]
KeyError: 'endfor'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./main/views.py", line 56, in result
    return render(request, 'main/result.html', context)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/shortcuts.py", line 19, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/backends/django.py", line 34, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/engine.py", line 143, in get_template
    template, origin = self.find_template(template_name)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/engine.py", line 125, in find_template
    template = loader.get_template(name, skip=skip)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/loaders/base.py", line 30, in get_template
    contents, origin, origin.template_name, self.engine,
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 155, in __init__
    self.nodelist = self.compile_nodelist()
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 193, in compile_nodelist
    return parser.parse()
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 478, in parse
    raise self.error(token, e)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 476, in parse
    compiled_result = compile_func(self, token)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/loader_tags.py", line 278, in do_extends
    nodelist = parser.parse()
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 478, in parse
    raise self.error(token, e)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 476, in parse
    compiled_result = compile_func(self, token)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/loader_tags.py", line 216, in do_block
    nodelist = parser.parse(('endblock',))
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 472, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 528, in invalid_block_tag
    get_text_list(["'%s'" % p for p in parse_until], 'or'),
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 50: 'endfor', expected 'endblock'. Did you forget to register or load this tag?
[pid: 6212|app: 0|req: 11/11] 172.17.0.1 () {60 vars in 4052 bytes} [Sun Jun  5 17:20:33 2022] GET /result/2 => generated 174896 bytes in 236 msecs (HTTP/1.1 500) 6 headers in 187 bytes (1 switches on core 0)
SIGINT/SIGTERM received...killing workers...
worker 1 buried after 1 seconds
goodbye to uWSGI.
VACUUM: pidfile removed.
VACUUM: unix socket /workspace/mbittest/tmp/MBIT.sock removed.
*** Starting uWSGI 2.0.20 (64bit) on [Sun Jun  5 17:20:45 2022] ***
compiled with version: 7.5.0 on 29 May 2022 17:23:40
os: Linux-4.4.0-1128-aws #142-Ubuntu SMP Fri Apr 16 12:42:33 UTC 2021
nodename: goorm
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 8
current working directory: /workspace/mbittest
writing pidfile to /workspace/mbittest/tmp/MBIT.pid
detected binary path: /workspace/mbittest/venv/bin/uwsgi
chdir() to /workspace/mbittest/
your memory page size is 4096 bytes
detected max file descriptor number: 1024
building mime-types dictionary from file /etc/mime.types...554 entry found
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /workspace/mbittest/tmp/MBIT.sock fd 3
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.7.4 (default, Nov  4 2020, 10:17:35)  [GCC 7.4.0]
PEP 405 virtualenv detected: /workspace/mbittest/venv/
Set PythonHome to /workspace/mbittest/venv/
Python main interpreter initialized at 0x559e6b428db0
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 154032 bytes (150 KB) for 1 cores
*** Operational MODE: single process ***
WSGI app 0 (mountpoint='') ready in 2 seconds on interpreter 0x559e6b428db0 pid: 6352 (default app)
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
spawned uWSGI master process (pid: 6352)
spawned uWSGI worker 1 (pid: 6355, cores: 1)
subprocess 6353 exited with code 0
Internal Server Error: /result/2
Traceback (most recent call last):
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 470, in parse
    compile_func = self.tags[command]
KeyError: 'endfor'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./main/views.py", line 56, in result
    return render(request, 'main/result.html', context)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/shortcuts.py", line 19, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/backends/django.py", line 34, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/engine.py", line 143, in get_template
    template, origin = self.find_template(template_name)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/engine.py", line 125, in find_template
    template = loader.get_template(name, skip=skip)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/loaders/base.py", line 30, in get_template
    contents, origin, origin.template_name, self.engine,
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 155, in __init__
    self.nodelist = self.compile_nodelist()
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 193, in compile_nodelist
    return parser.parse()
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 478, in parse
    raise self.error(token, e)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 476, in parse
    compiled_result = compile_func(self, token)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/loader_tags.py", line 278, in do_extends
    nodelist = parser.parse()
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 478, in parse
    raise self.error(token, e)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 476, in parse
    compiled_result = compile_func(self, token)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/loader_tags.py", line 216, in do_block
    nodelist = parser.parse(('endblock',))
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 472, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 528, in invalid_block_tag
    get_text_list(["'%s'" % p for p in parse_until], 'or'),
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 50: 'endfor', expected 'endblock'. Did you forget to register or load this tag?
[pid: 6355|app: 0|req: 1/1] 172.17.0.1 () {60 vars in 4075 bytes} [Sun Jun  5 17:20:51 2022] GET /result/2 => generated 175059 bytes in 361 msecs (HTTP/1.1 500) 6 headers in 187 bytes (1 switches on core 0)
[pid: 6355|app: 0|req: 2/2] 172.17.0.1 () {58 vars in 4033 bytes} [Sun Jun  5 17:21:22 2022] GET / => generated 2193 bytes in 43 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 6355|app: 0|req: 3/3] 172.17.0.1 () {58 vars in 4056 bytes} [Sun Jun  5 17:21:23 2022] GET /form/ => generated 34961 bytes in 298 msecs (HTTP/1.1 200) 7 headers in 350 bytes (1 switches on core 0)
[pid: 6355|app: 0|req: 4/4] 172.17.0.1 () {58 vars in 4019 bytes} [Sun Jun  5 17:21:28 2022] GET /form => generated 0 bytes in 1 msecs (HTTP/1.1 301) 5 headers in 174 bytes (1 switches on core 0)
Internal Server Error: /result/2
Traceback (most recent call last):
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 470, in parse
    compile_func = self.tags[command]
KeyError: 'endfor'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/core/handlers/base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./main/views.py", line 56, in result
    return render(request, 'main/result.html', context)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/shortcuts.py", line 19, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/backends/django.py", line 34, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/engine.py", line 143, in get_template
    template, origin = self.find_template(template_name)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/engine.py", line 125, in find_template
    template = loader.get_template(name, skip=skip)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/loaders/base.py", line 30, in get_template
    contents, origin, origin.template_name, self.engine,
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 155, in __init__
    self.nodelist = self.compile_nodelist()
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 193, in compile_nodelist
    return parser.parse()
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 478, in parse
    raise self.error(token, e)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 476, in parse
    compiled_result = compile_func(self, token)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/loader_tags.py", line 278, in do_extends
    nodelist = parser.parse()
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 478, in parse
    raise self.error(token, e)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 476, in parse
    compiled_result = compile_func(self, token)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/loader_tags.py", line 216, in do_block
    nodelist = parser.parse(('endblock',))
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 472, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/workspace/mbittest/venv/lib/python3.7/site-packages/django/template/base.py", line 528, in invalid_block_tag
    get_text_list(["'%s'" % p for p in parse_until], 'or'),
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 50: 'endfor', expected 'endblock'. Did you forget to register or load this tag?
[pid: 6355|app: 0|req: 5/5] 172.17.0.1 () {56 vars in 4003 bytes} [Sun Jun  5 17:21:32 2022] GET /result/2 => generated 174929 bytes in 283 msecs (HTTP/1.1 500) 6 headers in 187 bytes (1 switches on core 0)
[pid: 6355|app: 0|req: 6/6] 172.17.0.1 () {58 vars in 4011 bytes} [Sun Jun  5 17:22:12 2022] GET /result/2 => generated 7638 bytes in 8 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 6355|app: 0|req: 7/7] 172.17.0.1 () {58 vars in 3988 bytes} [Sun Jun  5 17:25:04 2022] GET /result/2 => generated 7999 bytes in 7 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
Not Found: /result/www.visitjeju.net
[pid: 6355|app: 0|req: 8/8] 172.17.0.1 () {58 vars in 4056 bytes} [Sun Jun  5 17:25:17 2022] GET /result/www.visitjeju.net => generated 2848 bytes in 7 msecs (HTTP/1.1 404) 5 headers in 159 bytes (1 switches on core 0)
Not Found: /result/www.visitjeju.net
[pid: 6355|app: 0|req: 9/9] 172.17.0.1 () {60 vars in 4087 bytes} [Sun Jun  5 17:27:46 2022] GET /result/www.visitjeju.net => generated 2848 bytes in 7 msecs (HTTP/1.1 404) 5 headers in 159 bytes (1 switches on core 0)
[pid: 6355|app: 0|req: 10/10] 172.17.0.1 () {58 vars in 3988 bytes} [Sun Jun  5 17:27:49 2022] GET /result/2 => generated 8010 bytes in 9 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
SIGINT/SIGTERM received...killing workers...
worker 1 buried after 1 seconds
goodbye to uWSGI.
VACUUM: pidfile removed.
VACUUM: unix socket /workspace/mbittest/tmp/MBIT.sock removed.
*** Starting uWSGI 2.0.20 (64bit) on [Sun Jun  5 17:35:58 2022] ***
compiled with version: 7.5.0 on 29 May 2022 17:23:40
os: Linux-4.4.0-1128-aws #142-Ubuntu SMP Fri Apr 16 12:42:33 UTC 2021
nodename: goorm
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 8
current working directory: /workspace/mbittest
writing pidfile to /workspace/mbittest/tmp/MBIT.pid
detected binary path: /workspace/mbittest/venv/bin/uwsgi
chdir() to /workspace/mbittest/
your memory page size is 4096 bytes
detected max file descriptor number: 1024
building mime-types dictionary from file /etc/mime.types...554 entry found
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /workspace/mbittest/tmp/MBIT.sock fd 3
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.7.4 (default, Nov  4 2020, 10:17:35)  [GCC 7.4.0]
PEP 405 virtualenv detected: /workspace/mbittest/venv/
Set PythonHome to /workspace/mbittest/venv/
Python main interpreter initialized at 0x56376d78adb0
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 154032 bytes (150 KB) for 1 cores
*** Operational MODE: single process ***
WSGI app 0 (mountpoint='') ready in 2 seconds on interpreter 0x56376d78adb0 pid: 6836 (default app)
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
spawned uWSGI master process (pid: 6836)
spawned uWSGI worker 1 (pid: 6839, cores: 1)
subprocess 6837 exited with code 0
[pid: 6839|app: 0|req: 1/1] 172.17.0.1 () {58 vars in 4034 bytes} [Sun Jun  5 17:36:01 2022] GET /result/2 => generated 8010 bytes in 216 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 6839|app: 0|req: 2/2] 172.17.0.1 () {58 vars in 3995 bytes} [Sun Jun  5 17:39:38 2022] GET /result/2 => generated 8022 bytes in 59 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 6839|app: 0|req: 3/3] 172.17.0.1 () {58 vars in 4032 bytes} [Sun Jun  5 17:39:43 2022] GET /all_results/ => generated 1520 bytes in 3 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
SIGINT/SIGTERM received...killing workers...
worker 1 buried after 1 seconds
goodbye to uWSGI.
VACUUM: pidfile removed.
VACUUM: unix socket /workspace/mbittest/tmp/MBIT.sock removed.
*** Starting uWSGI 2.0.20 (64bit) on [Sun Jun  5 17:58:33 2022] ***
compiled with version: 7.5.0 on 29 May 2022 17:23:40
os: Linux-4.4.0-1128-aws #142-Ubuntu SMP Fri Apr 16 12:42:33 UTC 2021
nodename: goorm
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 8
current working directory: /workspace/mbittest
writing pidfile to /workspace/mbittest/tmp/MBIT.pid
detected binary path: /workspace/mbittest/venv/bin/uwsgi
chdir() to /workspace/mbittest/
your memory page size is 4096 bytes
detected max file descriptor number: 1024
building mime-types dictionary from file /etc/mime.types...554 entry found
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to UNIX address /workspace/mbittest/tmp/MBIT.sock fd 3
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.7.4 (default, Nov  4 2020, 10:17:35)  [GCC 7.4.0]
PEP 405 virtualenv detected: /workspace/mbittest/venv/
Set PythonHome to /workspace/mbittest/venv/
Python main interpreter initialized at 0x560fc729a140
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 154032 bytes (150 KB) for 1 cores
*** Operational MODE: single process ***
WSGI app 0 (mountpoint='') ready in 2 seconds on interpreter 0x560fc729a140 pid: 647 (default app)
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
spawned uWSGI master process (pid: 647)
spawned uWSGI worker 1 (pid: 650, cores: 1)
subprocess 648 exited with code 0
[pid: 650|app: 0|req: 1/1] 172.17.0.1 () {58 vars in 4202 bytes} [Sun Jun  5 17:58:42 2022] GET / => generated 2193 bytes in 205 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 650|app: 0|req: 2/2] 172.17.0.1 () {58 vars in 4196 bytes} [Sun Jun  5 17:58:49 2022] GET /result/2 => generated 8022 bytes in 11 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 650|app: 0|req: 3/3] 172.17.0.1 () {58 vars in 4247 bytes} [Sun Jun  5 17:58:54 2022] GET /all_results/ => generated 1520 bytes in 4 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
Not Found: /result/2/
[pid: 650|app: 0|req: 4/4] 172.17.0.1 () {58 vars in 4199 bytes} [Sun Jun  5 18:00:08 2022] GET /result/2/ => generated 2982 bytes in 46 msecs (HTTP/1.1 404) 5 headers in 159 bytes (1 switches on core 0)
[pid: 650|app: 0|req: 5/5] 172.17.0.1 () {60 vars in 4232 bytes} [Sun Jun  5 18:01:00 2022] GET /all_results/ => generated 1520 bytes in 4 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 650|app: 0|req: 6/6] 172.17.0.1 () {58 vars in 4157 bytes} [Sun Jun  5 18:01:03 2022] GET /result/2 => generated 8051 bytes in 24 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 650|app: 0|req: 7/7] 172.17.0.1 () {58 vars in 4157 bytes} [Sun Jun  5 18:02:12 2022] GET /result/2 => generated 8084 bytes in 26 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 650|app: 0|req: 8/8] 172.17.0.1 () {58 vars in 4180 bytes} [Sun Jun  5 18:04:04 2022] GET /result/2 => generated 8084 bytes in 66 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 650|app: 0|req: 9/9] 172.17.0.1 () {58 vars in 4157 bytes} [Sun Jun  5 18:04:47 2022] GET /result/2 => generated 8126 bytes in 10 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 650|app: 0|req: 10/10] 172.17.0.1 () {58 vars in 4157 bytes} [Sun Jun  5 18:05:30 2022] GET /result/2 => generated 8119 bytes in 11 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 650|app: 0|req: 11/11] 172.17.0.1 () {58 vars in 4201 bytes} [Sun Jun  5 18:05:31 2022] GET /all_results/ => generated 1520 bytes in 4 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
Not Found: /result/1/
[pid: 650|app: 0|req: 12/12] 172.17.0.1 () {58 vars in 4199 bytes} [Sun Jun  5 18:06:11 2022] GET /result/1/ => generated 2982 bytes in 9 msecs (HTTP/1.1 404) 5 headers in 159 bytes (1 switches on core 0)
[pid: 650|app: 0|req: 13/13] 172.17.0.1 () {56 vars in 4126 bytes} [Sun Jun  5 18:06:15 2022] GET /result/1 => generated 8253 bytes in 15 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 650|app: 0|req: 14/14] 172.17.0.1 () {58 vars in 4157 bytes} [Sun Jun  5 18:06:47 2022] GET /result/1 => generated 8253 bytes in 100 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
Not Found: /result/2/
[pid: 650|app: 0|req: 15/15] 172.17.0.1 () {58 vars in 4199 bytes} [Sun Jun  5 18:06:51 2022] GET /result/2/ => generated 2982 bytes in 10 msecs (HTTP/1.1 404) 5 headers in 159 bytes (1 switches on core 0)
Not Found: /result/2/
[pid: 650|app: 0|req: 16/16] 172.17.0.1 () {60 vars in 4230 bytes} [Sun Jun  5 18:06:54 2022] GET /result/2/ => generated 2982 bytes in 9 msecs (HTTP/1.1 404) 5 headers in 159 bytes (1 switches on core 0)
[pid: 650|app: 0|req: 17/17] 172.17.0.1 () {58 vars in 4150 bytes} [Sun Jun  5 18:06:58 2022] GET /result/2 => generated 8119 bytes in 16 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 650|app: 0|req: 18/18] 172.17.0.1 () {58 vars in 4180 bytes} [Sun Jun  5 18:08:00 2022] GET /result/2 => generated 8119 bytes in 69 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 650|app: 0|req: 19/19] 172.17.0.1 () {58 vars in 4224 bytes} [Sun Jun  5 18:08:10 2022] GET /all_results/ => generated 1515 bytes in 4 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 650|app: 0|req: 20/20] 172.17.0.1 () {58 vars in 4220 bytes} [Sun Jun  5 18:08:42 2022] GET /result/1 => generated 8253 bytes in 10 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 650|app: 0|req: 21/21] 172.17.0.1 () {60 vars in 4228 bytes} [Sun Jun  5 18:14:56 2022] GET /result/1 => generated 8253 bytes in 16 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 650|app: 0|req: 22/22] 172.17.0.1 () {60 vars in 4228 bytes} [Sun Jun  5 18:18:58 2022] GET /result/1 => generated 8253 bytes in 22 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 650|app: 0|req: 23/23] 172.17.0.1 () {60 vars in 4228 bytes} [Sun Jun  5 18:19:04 2022] GET /result/1 => generated 8253 bytes in 11 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 650|app: 0|req: 24/24] 172.17.0.1 () {60 vars in 4228 bytes} [Sun Jun  5 18:25:39 2022] GET /result/1 => generated 8253 bytes in 37 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 650|app: 0|req: 25/25] 172.17.0.1 () {60 vars in 4228 bytes} [Sun Jun  5 18:27:46 2022] GET /result/1 => generated 8253 bytes in 40 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 650|app: 0|req: 26/26] 172.17.0.1 () {60 vars in 4228 bytes} [Sun Jun  5 18:29:33 2022] GET /result/1 => generated 8253 bytes in 42 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 650|app: 0|req: 27/27] 172.17.0.1 () {60 vars in 4228 bytes} [Sun Jun  5 18:31:36 2022] GET /result/1 => generated 8253 bytes in 21 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 650|app: 0|req: 28/28] 172.17.0.1 () {60 vars in 4228 bytes} [Sun Jun  5 18:38:17 2022] GET /result/1 => generated 8253 bytes in 97 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 650|app: 0|req: 29/29] 172.17.0.1 () {60 vars in 4228 bytes} [Sun Jun  5 18:41:38 2022] GET /result/1 => generated 8253 bytes in 97 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 650|app: 0|req: 30/30] 172.17.0.1 () {60 vars in 4228 bytes} [Sun Jun  5 18:43:27 2022] GET /result/1 => generated 8253 bytes in 72 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 650|app: 0|req: 31/31] 172.17.0.1 () {60 vars in 4228 bytes} [Sun Jun  5 18:47:36 2022] GET /result/1 => generated 8253 bytes in 21 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 650|app: 0|req: 32/32] 172.17.0.1 () {60 vars in 4228 bytes} [Sun Jun  5 18:47:38 2022] GET /result/1 => generated 8253 bytes in 13 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 650|app: 0|req: 33/33] 172.17.0.1 () {60 vars in 4251 bytes} [Sun Jun  5 18:48:15 2022] GET /result/1 => generated 8253 bytes in 102 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 650|app: 0|req: 34/34] 172.17.0.1 () {60 vars in 4251 bytes} [Sun Jun  5 18:52:09 2022] GET /result/1 => generated 8265 bytes in 105 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 650|app: 0|req: 35/35] 172.17.0.1 () {60 vars in 4228 bytes} [Sun Jun  5 18:53:16 2022] GET /result/1 => generated 8265 bytes in 10 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 650|app: 0|req: 36/36] 172.17.0.1 () {60 vars in 4228 bytes} [Sun Jun  5 19:01:32 2022] GET /result/1 => generated 8265 bytes in 10 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 650|app: 0|req: 37/37] 172.17.0.1 () {38 vars in 667 bytes} [Sun Jun  5 19:02:54 2022] GET / => generated 2193 bytes in 7 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 650|app: 0|req: 38/38] 172.17.0.1 () {38 vars in 683 bytes} [Sun Jun  5 19:02:58 2022] GET /result/1 => generated 8265 bytes in 87 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
[pid: 650|app: 0|req: 39/39] 172.17.0.1 () {38 vars in 667 bytes} [Sun Jun  5 19:03:05 2022] GET / => generated 2193 bytes in 14 msecs (HTTP/1.1 200) 5 headers in 167 bytes (1 switches on core 0)
SIGINT/SIGTERM received...killing workers...
worker 1 buried after 1 seconds
goodbye to uWSGI.
VACUUM: pidfile removed.
VACUUM: unix socket /workspace/mbittest/tmp/MBIT.sock removed.
